components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -7,6 +7,18 @@
     <string name="mozac_feature_addons_permissions_privacy_description">Read and modify privacy settings</string>
     <!-- Description for all_urls add-on permission. -->
     <string name="mozac_feature_addons_permissions_all_urls_description">Access your data for all websites</string>
+    <!-- Description for one site access add-on permission. %1$s  will be replaced by the DNS host name for which a web extension is requesting access (e.g., www.mozilla.org). -->
+    <string name="mozac_feature_addons_permissions_one_site_description">Access your data for %1$s</string>
+    <!-- Description for accessing multiple sites in one domain add-on permission. %1$s  will be replaced by the DNS domain for which a web extension is requesting access (e.g., mozilla.org). -->
+    <string name="mozac_feature_addons_permissions_sites_in_domain_description">Access your data for sites in the %1$s domain</string>
+    <!-- Description for accessing multiple sites permission. %1$d will be replaced by an integer indicating the number of additional hosts for which this webextension is requesting permission. -->
+    <string name="mozac_feature_addons_permissions_to_many_sites_description" tools:ignore="PluralsCandidate">Access your data on %1$d other site</string>
```

Comment:
NIT: Maybe add _single so it's easier for translation? Same with others below

Reply:
👍